sql/analyzer/declare.go

```diff
@@ -48,30 +49,41 @@ type declarationCursor struct {
 type declarationScope struct {
 	nextVarId  *int
 	nextCurId  *int
+	nextHandId *int
 	parent     *declarationScope
 	conditions map[string]*plan.DeclareCondition
 	variables  map[string]*declarationVariable
 	cursors    map[string]*declarationCursor
+	loops      map[string]struct{}
+
+	//TODO: implement proper handler support
```

Comment:
Can we get at least one skipped test where there are TODOs for functionality?

Reply:
I'mma work on completing stored procedures soon, so no point for now.